src/components/CountrySelector/PoACountrySelector.tsx

```diff
@@ -22,6 +21,15 @@ export type PoaProps = {
   StepComponentBaseProps
 
 class CountrySelection extends CountrySelectionBase {
```

Comment:
You might want to add types for the sate here.

Reply:
The state type ultimately inherits from the state type provided to the Component class by CountrySelectionBase. So, I'm not sure how to add a state type here without making CountrySelectionBase generic. If that's what you're suggestion, then I can easily make that change, though.